src/Features/LanguageServer/Protocol/Handler/Configuration/DidChangeConfigurationNotificationHandler.cs

```diff
@@ -67,6 +71,8 @@ public Task HandleNotificationAsync(DidChangeConfigurationParams request, Reques
 
         private async Task RefreshOptionsAsync(CancellationToken cancellationToken)
         {
+            using var _ = _listener.BeginAsyncOperation(nameof(DidChangeConfigurationNotificationHandler));
```

Comment:
❓ Why would this be necessary. It appears that all callers of this method await the result, so it does not appear to be possible for this method to be invoked in a fire-and-forget manner.

📝 Asynchronous operation listeners are added at the point where a caller will start an asynchronous operation and then not wait for that operation to complete. This location does not match that characteristic.